testsuite/MDAnalysisTests/coordinates/test_crd.py

```diff
@@ -66,11 +66,13 @@ def CRD_iter(fn):
         for ref, other in zip(CRD_iter(CRD), CRD_iter(outfile)):
             assert ref == other
 
-    def test_write_EXT(self):
-        # TODO: Write tests that use EXT output format
-        # Must have *lots* of atoms, maybe fake the system
-        # to make tests faster
-        pass
+    def test_write_EXT(self, u, outfile):
+        # Use the `extended` keyword to force the EXT format
+        try:
+            u.atoms.write(outfile, extended=True)
+        except Exception:
```

Comment:
Allowing a test to fail "naturally" is probably the right thing to do instead of capturing all possible exceptions and then re-raising (there's no real advantage to the generic capture, and it may even make the failure harder to diagnose).

Some other thoughts here:

1) The test as written would produce an artifact in the user/developer space because it is not scoped to a temporary directory. There are examples of using the `tmpdir` `pytest` fixture in this module, so that the test files get written without leaving artifacts behind. If you do use the examples in this testing module as inspiration, I'd suggest avoiding the construct with `outfile = str(tmpdir) + '/out.crd' ` and favoring something like `os.path.join` to compose the path in a platform-portable way (we should probably fix that, but it is orthogonal to this PR)
2) Is there anything you could check/verify in the output file that gets written? For example, some string/binary data that is unique to the `EXT` format that would signal that MDAnalysis has done the right thing? Making an assertion on that would make the test much more robust.

Reply:
@tylerjereddy Thanks for your comments. I used `test_pdb.py` as inspiration to how the `outfile` variable is defined. Let me know if that solves the issue. 

Also, CRD files in extended format are expected to have a `EXT` string in the third line. I made an assertion for that existence that ran just fine locally. I hope that enhances the test.